deployments/base/usdc/configuration.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "Compound USDC",
+  "symbol": "cUSDCv3",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913",
+  "baseTokenPriceFeed": "0x7e860098F58bBFC8648a4311b374B1D669a2bc6B",
+  "borrowMin": "1e0",
+  "pauseGuardian": "0x3cb4653f3b45f448d9100b118b75a1503281d2ee",
+  "storeFrontPriceFactor": 0.6,
+  "targetReserves": "5000000e6",
+  "rates": {
+    "supplyKink": 0.8,
+    "supplySlopeLow": 0.0325,
+    "supplySlopeHigh": 0.4,
+    "supplyBase": 0,
+    "borrowKink": 0.8,
+    "borrowSlopeLow": 0.035,
+    "borrowSlopeHigh": 0.25,
+    "borrowBase": 0.015
```

Comment:
These should mirror the latest IR params introduced in this proposal: https://compound.finance/governance/proposals/207

Reply:
Yes good call